<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskswindow</class>
 <widget class="QDialog" name="taskswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Lato</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tasks</string>
  </property>
  <widget class="QScrollArea" name="tasksArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="taskAreaWidgets">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>639</width>
      <height>539</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="taskBox">
       <property name="minimumSize">
        <size>
         <width>597</width>
         <height>180</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="taskTitle">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>431</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tytuł zadania</string>
        </property>
       </widget>
       <widget class="QLabel" name="taskDesc">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>521</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Opis zadania</string>
        </property>
       </widget>
       <widget class="QLabel" name="taskDate">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>120</y>
          <width>391</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>12.12.2022</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nadchodzące wydarzenia</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Dodaj wydarzenie</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="addDate">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>410</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>490</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addTitle">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>180</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wydarzenie</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="addDescription">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>280</y>
     <width>401</width>
     <height>85</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>250</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Opis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>752</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="communicatLabel">
   <property name="geometry">
    <rect>
     <x>772</x>
     <y>600</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>taskswindow</receiver>
   <slot>onAddButton(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>917</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddButton(bool)</slot>
 </slots>
</ui>
